src/components/timelapse/DatePicker.vue

```diff
@@ -1,22 +1,23 @@
 <template>
   <div>
     <div class="datepicker-trigger">
-      <v-icon id="event-icon" large color="teal">event</v-icon>
-      <input
-        type="text"
-        id="datepicker-trigger"
-        prepend-inner-icon="event"
-        placeholder="Select dates"
-        :value="formatDates(dateOne, dateTwo)"
-      />
-
+      <div>
+        <img src="../../../public/date_icon.png" id="input_img"/>
```

Comment:
Instead of an image, can we use an icon? Something like:
```html
<i class="material-icons">event</i>
```
Or:
```html
<i class="material-icons">calendar_today</i>
```
You can override the default CSS for this icon by either specifying an `id` or another `class`.